zrml/authorized/src/lib.rs

```diff
@@ -181,6 +159,12 @@ mod pallet {
         T::AccountId,
         OutcomeReport,
     >;
+
+    /// Maps the market id to the outcome reported by the authorized account.    
+    #[pallet::storage]
+    #[pallet::getter(fn outcomes)]
+    pub type AuthorizedOutcomeReports<T: Config> =
+        StorageMap<_, Blake2_128Concat, MarketIdOf<T>, OutcomeReport, OptionQuery>;
```

Comment:
We can use the significantly faster and more light-weight hasher [xx64](https://docs.substrate.io/rustdocs/latest/frame_support/struct.Twox64Concat.html). Since the users don't have direct control over the keys, a cryptographic hash function like `Blake2_128` is not necessary.

Reply:
Fixed in [868f737](https://github.com/zeitgeistpm/zeitgeist/pull/574/commits/868f73725af190be47d052930efac7669fbe91a2)